lib/control/validation/content.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const HTTP_BAD_REQUEST = 400;
+
+const M_INSTANCE_ID = /^i-[a-f0-9]{8,}$/;
```

Comment:
The regex is only checking to make sure that the ID has at least 8, s no change will be needed when Amazon updates their IDs.
However, once all IDs have at least a length of 12, this should be updated.